release notes/v0.37.0.md

```diff
@@ -0,0 +1,72 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
+
+## New Features!
+
+### Added support for local files to --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+Added file hook for a logger with the possibility to define a minimal reporting level.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added the basic functionality of the JavaScript event loops ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step that is limited by a `setTimeout` method as an example. We advise avoiding using it in production code. But for sure, we'll continue to work on extending that feature in the next k6's releases.
```

Comment:
```suggestion
We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions. 

If you are an extension developer please use it and give your feedback. But take into consideration that it's likely that the API may change
```

We don't want to advertise the setTimeout or `k6/experimental` *at all*. It only exists so we can test the event loop - otherwise we would've had removed it.